server/src/main/java/keywhiz/service/daos/SecretSeriesDAO.java

```diff
@@ -348,8 +368,10 @@ public void deleteSecretSeriesById(long id) {
   }
 
   public void renameSecretSeriesById(long secretId, String name, String creator, long now) {
+    String rowHmac = computeRowHmac(secretId, name);
     dslContext.update(SECRETS)
         .set(SECRETS.NAME, name)
+        .set(SECRETS.ROW_HMAC, rowHmac)
         .set(SECRETS.UPDATEDBY, creator)
         .set(SECRETS.UPDATEDAT, now)
         .where(SECRETS.ID.eq(secretId))
```

Comment:
This is our bug fix: the row HMAC is now calculated and updated as part of the renaming process.